llvm/unittests/IR/DebugInfoTest.cpp

```diff
@@ -1301,4 +1301,33 @@ TEST(DIBuilder, CompositeTypes) {
   EXPECT_EQ(Enum->getTag(), dwarf::DW_TAG_enumeration_type);
 }
 
+TEST(DIBuilder, DynamicOffsetAndSize) {
+  LLVMContext Ctx;
+  std::unique_ptr<Module> M(new Module("MyModule", Ctx));
```

Comment:
Prefer `std::make_unique` and optionally omit the type on the LHS since it'll be clear from the RHS, like this:
```
auto M = std::make_unique<Module>("MyModule", Ctx);
```

Reply:
Fixed in the next version.